### DISCUSSION TITLE
Notebook para trabalho e jogos.

### DISCUSSION THREAD

**Discussion by andrelsa** [frontendbr/forum#1348] [Perguntas e Respostas]
Gostaria de ajuda, busco um notebook ate 4,5k, para trabalho e jogar. Qual seria o melhor custo x beneficio no momento? Pesquisei alguns e ainda estou com dúvidas.

**Comment by felipefialho** [comment]
Linkando: #1285 #237

**Comment by dtfialho** [comment]
@andrelsa a minha única dica pra notebook para trabalhar e poder jogar, é, tomar cuidado na hora que estiver trabalhando, porque sempre vai ter aquele iconezinho lá que o dedo no mouse vai sempre coçar para clicar e acaba gerando distrações. Por outro lado se você conseguir resistir a tentação não tem problema. :)

**Comment by calaca** [comment]
Tenho um Dell Inspiron 15 Gaming que me serve muito bem para esses dois propósitos. O que tenho é o modelo i15-7567-a20p com mais 8 GB de memória. Creio que paguei algo por volta de 3,6K em dezembro do ano passado.
Já a solução para não ficar na tentação de jogar em horário de trabalho (como levantado pelo @dtfialho ) é simples: faça um dual boot com Windows e Linux. Windows apenas para lazer e Linux para trabalho.

Espero que ajude :)

**Comment by lempkilobyte** [comment]
Pode dar uma olhada nessa lista: https://kilobyte.com.br/top-notebooks-abaixo-de-4-mil/

**Comment by anotte** [comment]
Comprei um Samsung Odyssey 1050, i7 com 16Gb RAM que está me atendendo bem,
mas tenho que confessar que mais trabalho que jogo... :-(

Saiu por 4.6k e lembro de ter sido o melhor custo benefício. Unico upgrade
foi trocar o HD convencional por um SSD.

my two cents...

**Comment by VivianMoreno** [comment]
in 2021 if any of us are still playing among us

**Comment by VivianMoreno** [comment]
you can refer more here. I see a lot of games https://techgara.com/among-us

**Comment by giovannism20** [comment]
@felipefialho Spam [#1](https://github.com/frontendbr/forum/discussions/1348#discussioncomment-487577) [#2](https://github.com/frontendbr/forum/discussions/1348#discussioncomment-553293)
